src/shim.c

```diff
@@ -704,7 +704,7 @@ handle_proxy_notification(struct cc_shim *shim, struct frame *fr)
 				"to proxy at %s\n", shim->proxy_address);
 		}
 
-		shim_debug("Exit status for container: %d\n", code);
+		shim_debug("Exit status for container %s: %d\n", code);
```

Comment:
Have you missed the `shim->container_id` new/extra argument here?

Reply:
Good catch! I'll update...

Reply:
Fixed.